test/e2e/disruption.go

```diff
@@ -49,6 +50,11 @@ var _ = framework.KubeDescribe("DisruptionController", func() {
 	BeforeEach(func() {
 		cs = f.StagingClient
 		ns = f.Namespace.Name
+
+		// this test wants powerful permissions.  Since the namespace names are unique, we can leave this
+		// lying around so we don't have to race any caches
+		framework.BindClusterRole(f.ClientSet.Rbac(), "podsecuritypolicy:privileged", f.Namespace.Name,
```

Comment:
this role will only exist in GCE with PSP enabled... what happens when the role is not present?